src/scrapers/platformen/mijnbuurtje.py

```diff
@@ -0,0 +1,180 @@
+
+                output = initiative_parser.apply_schemas()
+                for uri in output['initiatives']:
+                    yield InitiativeImport(source_uri=uri)
+
+                page_counter = page_counter + 1
+        except Exception as ex:
+            raise ScrapeException("Error loading list of initiatives") from ex
+
+    def complete(self, initiative: InitiativeImport):
+        try:
+            session_metadata = self.item_parser.get_session_metadata(initiative.source_uri)
+            full_initiative = self.item_parser.apply_schemas(metadata=session_metadata,
+                                                             url=initiative.source_uri)
+            for key, value in full_initiative.items():
+                setattr(initiative, key, value)
+
+            if not initiative.location:
+                initiative.location = self.config.location
+        except Exception as ex:
+            raise ScrapeException(f"Error scraping {initiative.source_uri}") from ex
+
+    def find_initiative_links(self, elements):
+        """
+        filters links. Input seems to be 1 on 1 now. Extracted to method for easier debugging.
+        """
+        initiatives = []
+        for e in elements:
+            links = re.findall(self.config.details_endpoint + "\\d{4,5}", e.attrib['href'])
+            if len(links) > 0:
+                initiatives.append(links)
+
+        return initiatives
+
+    @staticmethod
+    def strip_text(element, strip: str):
+        if element is None:
+            return None
+        if not element.text:
+            return None
+
+        stripped = element.text.replace(strip, "")
+        return stripped
+
+    @staticmethod
+    def recursive_text(elements):
+        texts = []
+        for e in elements:
+            if type(e) in [etree._ElementUnicodeResult]:
+                texts.append(f"{e.strip()}")
+            else:
+                texts.append(f"{e.text.strip()}")
+        return ' '.join(texts)
+
+    @staticmethod
+    def format_group(original_group: str):
+        if len(re.findall("Hulpvraag.*?", original_group)) > 0:
+            return 'demand'
+        else:
+            return 'supply'
+
+    @staticmethod
+    def format_organizer(organizer):
+        if organizer is not None:
+            organizer = re.sub('-', ' ', organizer)
+            name_segments = organizer.split()
+            organizer_name = None if len(name_segments) == 0 else name_segments[0]
+
+            return organizer_name
+
+    # I assume there's a better way using locales
+    MONTHS = ["januari", "februari", "maart", "april", "mei", "juni", "juli",
+              "augustus", "september", "oktober", "november", "december"]
+
+    @staticmethod
+    def format_date(date_str: str):
+        # TODO: This is an important field. Should break here!
+        if not date_str:
+            raise ValueError("Always expecting a date string for conversion!")
+
+        segments = date_str.split()
```

Comment:
Als je zeker weet dat het er 3 zijn, dan mag je ook dit doen:
```
         day, month, year = date_str.split()
```

Reply:
Kans is groot dat het zo is, maar als ze in ene besluiten er een tijd bij te zetten oud dan klopt het al niet meer.

Reply:
Ah wait, anders krijg je die not enough values to unpack ValueError indd. Dan zou ik dat ook kunnen gebruiken. Hoewel mijn foutmelding wel duidelijker is. Kan dit alsnog wel catchen rethrowen met deze foutmelding met als voordeel dat ik dan indd wel leesbaardere variabelenamen heb.